src/Generator.cpp

```diff
@@ -115,6 +115,26 @@ int generate_filter_main(int argc, char **argv, std::ostream &cerr) {
         cerr << kUsage;
         return 1;
```

Comment:
Could use split_string(flags_info["-e"], ",") (from Util.cpp) instead of manually parsing here.